core/src/main/scala/kafka/log/LogConfig.scala

```diff
@@ -109,7 +109,7 @@ object LogConfig {
   val FlushIntervalDoc = "The number of messages that can be written to the log before a flush is forced"
   val FlushMsDoc = "The amount of time the log can have dirty data before a flush is forced"
   val RetentionSizeDoc = "The approximate total number of bytes this log can use"
-  val RetentionMsDoc = "The approximate maximum age of the last segment that is retained"
+  val RetentionMsDoc = "The approximate maximum number of milli seconds of the last segment that is retained."
```

Comment:
`milliseconds` is one word and it seems like we are not using a full stop at the end of each doc description in the other cases